### DISCUSSION TITLE
The Craft 4 error notification is too easily missed

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#12591] [General]
In Craft 3, when you saved an entry and there was a problem that prevented it from saving (like missing a required field), a very noticeable red bar appeared at the top of the screen saying it couldn't be saved.

![image](https://user-images.githubusercontent.com/1250017/215255880-1e89772c-61fe-4aac-a3e0-9416f3fcc0f2.png)

In Craft 4, this notification is white, unobtrusive, and appears in the bottom left of the screen (on the far opposite side to the "Save" button). I miss it practically every time, and so do our designers and clients. Which means they ask me why the CMS isn't working and it won't save their entry.

![image](https://user-images.githubusercontent.com/1250017/215255871-608bdc44-2d86-441a-a880-c72f430967b2.png)

Would you please consider either / both moving these notifications back to the top and making them much more visually obvious?

**Comment by wsydney76** [comment]
So true.

Fortunately, you can restore the previous behavior and customize it to your liking with a little CSS:

https://github.com/wsydney76/craft4-ddev-starter/blob/main/modules/main/resources/dist/cpstyles.css#L33

/Mel

**Reply by darylknight** [reply]
This is great, thank you :) I still think the default should be way more obvious from Craft though - not just people who use a plugin to override the CSS

**Comment by brandonkelly** [comment]
We’re working to make element validation errors significantly better with #12125. Guessing that will resolve this.
